front/app/containers/Authentication/Modal.tsx

```diff
@@ -270,6 +270,7 @@ const AuthModal = ({ setModalOpen }: Props) => {
             onSwitchFlow={transition(currentStep, 'SWITCH_FLOW')}
             onGoBack={transition(currentStep, 'GO_BACK')}
             onSubmit={transition(currentStep, 'SIGN_IN')}
+            closeModal={handleClose}
```

Comment:
```suggestion
            closeModal={transition(currentStep, 'CLOSE')}
```
This way you have TS support to make sure that `CLOSE` is a valid action at your current step